usr/lib/common/object.c

```diff
@@ -143,8 +143,7 @@ CK_RV object_copy(STDLL_TokData_t * tokdata,
                   CK_ATTRIBUTE * pTemplate,
                   CK_ULONG ulCount, OBJECT * old_obj, OBJECT ** new_obj)
 {
-    TEMPLATE *tmpl = NULL;
-    TEMPLATE *new_tmpl = NULL;
+    TEMPLATE *tmpl, *new_tmpl = NULL;
```

Comment:
As both variables, tmpl, and new_tmpl are unconditionally assigned in lines 158 and 159, initializing new_tmpl = NULL is not necessary, but also not wrong. However, I would either initialize both variables or none to avoid confusion.

Reply:
yes, right, initializing don't make sense here. Will fix this.